src/implementations/config/create.ts

```diff
@@ -0,0 +1,211 @@
+  const documentStoreAddress = await generateDocumentStoreOrTokenRegistryOrDid(
+    typesOfForms,
+    "VERIFIABLE_DOCUMENT",
+    "DNS-TXT"
+  );
+  const verifiableDocumentDnsTxtName = documentStoreAddress
+    ? await createTemporaryDns({ networkId: 3, address: documentStoreAddress, sandboxEndpoint: sandboxEndpointUrl })
+    : "";
+  const verifiableDocumentDnsDidName = await generateDocumentStoreOrTokenRegistryOrDid(
+    typesOfForms,
+    "VERIFIABLE_DOCUMENT",
+    "DNS-DID"
+  );
+  const tokenRegistryAddress = await generateDocumentStoreOrTokenRegistryOrDid(
+    typesOfForms,
+    "TRANSFERABLE_RECORD",
+    "DNS-TXT"
+  );
+  const tokenRegistryDnsName = tokenRegistryAddress
+    ? await createTemporaryDns({ networkId: 3, address: tokenRegistryAddress, sandboxEndpoint: sandboxEndpointUrl })
+    : "";
+
+  // replace the values in the forms with the updated value
+  formsInTemplate.forEach((form) => {
+    if (form.type === "VERIFIABLE_DOCUMENT") {
+      const updatedIssuers = form.defaults.issuers.map((issuer: Issuer) => {
+        if (issuer.identityProof?.type === "DNS-TXT") {
+          issuer.name = "DEMO DOCUMENT STORE";
+          issuer.documentStore = documentStoreAddress;
+          if (issuer.identityProof?.location) issuer.identityProof.location = verifiableDocumentDnsTxtName;
+        } else if (issuer.identityProof?.type === "DNS-DID") {
+          issuer.name = "DEMO ISSUER";
+          issuer.id = `did:ethr:0x${walletObject.address}`;
+          if (issuer.revocation?.type) issuer.revocation.type = "NONE" as RevocationType;
+          if (issuer.identityProof?.location) issuer.identityProof.location = verifiableDocumentDnsDidName;
+          if (issuer.identityProof?.key) issuer.identityProof.key = `did:ethr:0x${walletObject.address}#controller`;
+        }
+        return issuer;
+      });
+      form.defaults.issuers = updatedIssuers;
+    }
+    if (form.type === "TRANSFERABLE_RECORD") {
+      const updatedIssuers = form.defaults.issuers.map((issuer: Issuer) => {
+        issuer.name = "DEMO TOKEN REGISTRY";
+        issuer.tokenRegistry = tokenRegistryAddress;
+        if (issuer.identityProof?.location) issuer.identityProof.location = tokenRegistryDnsName;
+        return issuer;
+      });
+      form.defaults.issuers = updatedIssuers;
+    }
+  });
+
+  configFile.forms = formsInTemplate;
+  const configFileName = "config.json";
+  const outputPath = path.join(outputDir, configFileName);
+  fs.writeFileSync(outputPath, JSON.stringify(configFile, null, 2));
+  return outputPath;
+  // success(`Config file successfully generated at ${highlight(`${outputDir}${configFileName}`)}`);
+};
+
+const selectTemplateFile = async (configType: string, configTemplatePath: string): Promise<ConfigFile> => {
+  switch (configType) {
+    case "tradetrust":
+      return tradetrustConfig as ConfigFile;
+
+    default:
+      return JSON.parse(await readFile(configTemplatePath));
+  }
+};
```

Comment:
think shouldn't mix the use of both `configType ` and `configTemplatePath` here

Reply:
y leh? what do u mean leh? cos what i am doing here is to check the user selection, cos u only can choose either using configType or configTemplatePath, so if u use configType we check which type of config file u want to generate, and if u dont select configType means u select configTemplatePath where u supply a config file and we do a duplicate of that config file and update the required fields to the ones that are generated.